CODE_OF_CONDUCT.md

```diff
@@ -58,11 +58,7 @@ If you believe someone has violated our Code of Conduct, we encourage you to rep
 
 Make a report to any of the maintainers listed in the ProtoSchool Denver Chapter [README.md](https://github.com/ProtoSchool/Denver/blob/master/README.md):
  
-**In the event of a conflict of interest, you are encouraged to directly contact any of other ProtoSchool Chaper Maintainers, [found here](https://github.com/ProtoSchool), or the Orginization's lead maintainers:**
-
-- [Teri Chadbourne](https://terichadbourne.com/)
-Community Manager at [ProtocolLabs](https://protocol.ai)
-[teri@protocol.ai](teri@protocol.ai)
+**In case of a conflict of interest with the Denver Chaper maintainers, or if you feel that this chapter's organizers are not living up to the standards laid out in our Code of Conduct, please follow the reporting procedures laid out in the broader [ProtoSchool Code of Conduct](https://github.com/ProtoSchool/organizing/blob/master/CODE_OF_CONDUCT.md) to reach the project's lead maintainers - [protoschool@protocol.ai](protoschool@protocol.ai)**
```

Comment:
"Denver Chaper" -> "Denver Chapter"

Reply:
The link created for protoschool@protocol.ai tries to go to https://github.com/ProtoSchool/Denver/blob/96cbf3cbf8391066e141f72a53ca7acf0da1cd6a/protoschool@protocol.ai and gets a 404. I presume you need to prepend `mailto:`.